content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -276,11 +281,15 @@ The `info` object may have some arrangement of the following fields:
 | `peers`                             | Number          | How many other `rippled` servers this one is currently connected to. |
 | `pubkey_node`                       | String          | Public key used to verify this server for peer-to-peer communications. This [_node key pair_](peer-protocol.html#node-key-pair) is automatically generated by the server the first time it starts up. (If deleted, the server can create a new pair of keys.) You can set a persistent value in the config file using the `[node_seed]` config option, which is useful for [clustering](clustering.html). |
 | `pubkey_validator`                  | String          | _(Admin only)_ Public key used by this node to sign ledger validations. This _validation key pair_ is derived from the `[validator_token]` or `[validation_seed]` config field. |
+| `reporting`                         | Object          | _([Reporting mode](rippled-server-modes.html) servers only)_ Information about this server's reporting-mode specific configurations. |
+| `reporting.etl_sources`             | Array           | _([Reporting mode](rippled-server-modes.html) servers only)_ A list of p2p-mode servers this reporting mode is retrieving data from. Each entry in this array is an [ETL Source object](#etl-source-object). |
+| `reporting.is_writer`               | Boolean         | _([Reporting mode](rippled-server-modes.html) servers only)_ If `true`, this server is configured to write to the relational database with ledger data. If `false`, it is configured read-only and relies on other reporting mode servers to populate the database. |
+| `reporting.last_publish_time`       | String          | _([Reporting mode](rippled-server-modes.html) servers only)_ An ISO 8601 timestamp indicating when this server last saw a new validated ledger from any of its p2p mode sources. |
```

Comment:
This is actually the last time the server published a ledger to its subscription streams. Doesn't have anything to do with p2p.